docs/setup/build.md

```diff
@@ -13,7 +13,7 @@ Build the Curium Project
         make mainnet
  
 3.  Ensure Bluzelle CLI and daemon work by executing the binaries, you should 
-    be able to execute the apps from your users directory:
+    be able to execute the apps from your home directory:
```

Comment:
Do you want to mention that the binaries are installed into the $GOPATH/bin directory and it should be in your $PATH?